### PR TITLE
added height

### PR DESCRIPTION
**Pull request by Joobee30**
JIRA: link to jira ticket
## Context:
What is the ticket about and why are we doing this change.
## Changes
What are the various changes and what other modules do those changes effect.
This can be bullet point or sentence format.
## Before and After UI (Required for UI-impacting PRs)
If this is a change to the UI, include before and after screenshots to show the differences.
If this is a new UI feature, include screenshots to show reviewers what it looks like. 
## Dev notes (Optional)
- Specific technical changes that should be noted
## Linked pull requests (Optional)
- pull request link

### CODE REVIEW

File: blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/DataLayer/Client.java [hunk 1]
```diff
@@ -21,4 +21,5 @@ public class Client {
     public String email;
     public String phoneNumber;
     public String address;
+    public double height;
```
Comment (dazhuox) [inline blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/DataLayer/Client.java:24]: Why did you add this? we dont need height for blackjack players.

File: blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/BusinessLayer/ClientService.java [hunk 2]
```diff
@@ -1,4 +1,5 @@
 package com.blackjacktrainer.blackjacktrainerbackend.User.BusinessLayer;
 
 public interface ClientService {
+
```
Comment (dazhuox) [inline blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/BusinessLayer/ClientService.java:4]: i like the spacing

**Review by dazhuox — APPROVED** [review]

File: blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/BusinessLayer/ClientService.java (hunk 2 above)
Comment (tongmaximewu) [reply blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/BusinessLayer/ClientService.java:4]: hi

File: blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/DataLayer/Client.java (hunk 1 above)
Comment (Joobee30) [reply blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/DataLayer/Client.java:24]: Ummmm actually you need height.

**Review by Rampex1 — APPROVED** [review]

File: blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/BusinessLayer/ClientService.java (hunk 2 above)
Comment (Joobee30) [reply blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/BusinessLayer/ClientService.java:4]: thanks cuh

**Review by tongmaximewu — APPROVED** [review]
nice
